README.md

```diff
@@ -63,6 +63,14 @@ There is nothing special to configure on PyFileMaker. Just make sure the
 PyFileMaker directory, which contains the file FMPro.py/FMServer.py file is
 somewhere in your Python path so that Python knows where to find the module.
 
+You can install v2.6 from pypi using pip
```

Comment:
Agreed. But 2.6 is the only version available via pypi. I did a little work and have 3.0.2.1 setup on the testpypi:

https://testpypi.python.org/pypi/PyFileMaker/3.0.2.1

Finish off the PR and then you can put the real 3.0.2.1 release up on pypi?

Don't know what to do about the README. Github want's markdown but pypi wants text.